packages/flutter/lib/src/material/drop_down.dart

```diff
@@ -247,6 +249,32 @@ class DropDownMenuItem<T> extends StatelessWidget {
   }
 }
 
+/// An inherited widget that causes any descendant [DropDownButton]
+/// widgets to not include their regular underline.
+///
+/// This is used by [DataTable] to remove the underline from any
+/// [DropDownButton] widgets placed within material data tables, as
+/// required by the material design specification.
+class DropDownButtonHideUnderline extends InheritedWidget {
+  /// Creates a [DropDownButtonHideUnderline]. A non-null [child] must
+  /// be given.
+  DropDownButtonHideUnderline({
+    Key key,
+    Widget child
+  }) : super(key: key, child: child) {
+    assert(child != null);
+  }
+
+  /// Returns whether the underline of [DropDownButton] widgets should
+  /// be hidden.
+  static bool here(BuildContext context) {
```

Comment:
The problem is then it would look odd in the place where you build it. I've changed 'here' to 'at' to match Focus.